cmd/endpoint-ellipses_test.go

```diff
@@ -472,6 +472,29 @@ func TestParseEndpointSet(t *testing.T) {
 			},
 			true,
 		},
+		// IPv6 ellipses expansion
+		{
+			"http://[2001:3984:3989::{1...10}]/disk{1...10}",
```

Comment:
In the IPv6 address `2001:3984:3989::xxx` each digit is hexadecimal. So looks like the above expansion would give a wrong sequence - the last hextet group in the last address in the sequence should be `a` and not `10`.

Reply:
while this is a valid issue, it may not be supportable from IPv6 point of view. Ellipses are opaque to input i.e it only expands what was asked. So this case has to be ignored for now - the assumption is direct IPv6 consumers are rare and most use DNS.

Reply:
Added support for hexadecimal expansion as well.